interface/src/project/SettingsApplication.tsx

```diff
@@ -404,7 +405,7 @@ const SettingsApplication: FC = () => {
           />
           {data.shower_alert && (
             <>
-              <Grid item xs={4}>
```

Comment:
ok to have larger boxes if you prefer, but the spacing between the two is large and it looks funny on a desktop browser

Reply:
my new fixes should solve the text clipping issues